src/benchmarklib/tpch/tpch_table_generator.cpp

```diff
@@ -105,14 +105,14 @@ void dbgen_cleanup() {
   }
 
   if (asc_date) {
+    // NOLINTBEGIN(cppcoreguidelines-no-malloc,hicpp-no-malloc,cppcoreguidelines-owning-memory)
```

Comment:
Might look confusing. New is the `bugprone-multi-level-implicit-pointer-conversion` exception, the rest is reformatting.